cpp/test/sg/decisiontree_batchedlevel_unittest.cu

```diff
@@ -28,6 +28,7 @@
 namespace ML {
 namespace DT {
 
+/*
```

Comment:
Is this test not needed anymore? I guess it will require significant rewrite as kerne calls etc are changed a lot. I am okay if we decide to drop this test.

Reply:
I think these cases are well covered by existing tests so I think dropping them is ok. I would like to write some more unit tests covering metrics specifically at some point.